src/renderer/components/menu/menu.tsx

```diff
@@ -319,28 +321,40 @@ export class Menu extends React.Component<MenuProps, State> {
       return item;
     });
 
-    const menu = (
-      <MenuContext.Provider value={this}>
+    let menu = (
+      <ul
+        id={id}
+        ref={this.bindRef}
+        className={className}
+        style={{
+          left: this.state?.menuStyle?.left,
+          top: this.state?.menuStyle?.top,
```

Comment:
`this.state` might not exists?

Reply:
Nope, just copy pasted it the old code. Probably should be cleaned up in the future.